integration/log.go

```diff
@@ -382,6 +390,21 @@ func checkInclusionProofTreeSizeOutOfRange(logID int64, client trillian.Trillian
 // proofs returned should match ones computed by the alternate Merkle Tree implementation, which differs
 // from what the log uses.
 func checkInclusionProofsAtIndex(index int64, logID int64, tree *merkle.InMemoryMerkleTree, client trillian.TrillianLogClient, params TestParameters) error {
+	// Get MerkleLeafHash to use in VerifyInclusionProof.
```

Comment:
I didn't see a local copy of the leaf hash. Please point it out if I'm missing it. 

In the meantime, I've adjusted the test to compute the MerkleLeafHash after fetching the leaf value.